.github/workflows/mirror-images.yml

```diff
@@ -12,11 +16,11 @@ jobs:
   mirror:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v3
```

Comment:
i bumped these this morning via dependabot, it seemed fine. lmk if they should actually be pinned to these versions.

Reply:
ugh, ok I'll revert, same for below